docs/pages/machine-id/access-guides/applications.mdx

```diff
@@ -1,204 +1,111 @@
 ---
 title: Machine ID with Application Access
-description: How to use Machine ID to access Applications
+description: How to use Machine ID to access applications
 ---
 
 (!docs/pages/includes/machine-id/v2-config-warning.mdx!)
 
-This guide demonstrates how to configure an automated service to use
-Machine ID to access a web application protected by Teleport.
+Teleport Application Access protects and controls access to HTTP and TCP
```

Comment:
```suggestion
Teleport protects and controls access to HTTP and TCP
```
So "Application Access" doesn't seem like a distinct product (we're not positioning it as one anymore).

Reply:
Wasn't aware of this change - thank you.